README.md

```diff
@@ -1,2 +1,25 @@
 # Seine-HelloWorld
 The testing repo for Seine Bot
+
+
+# Assignment 1 (5/19/24) Test Event Types for Seine AI
+
+1. Fork this repository. Create/Edit a file of your choice.
+2. Create a PR with your commmited changes
+3. Test out the following event types:
+ - add PR Request review
+ - specific line comment review 
+
+Demos below have more details:
+
+*add PR request review should automatically run when you make a PR*
+
+after you push your commits. on github (or remotely) create a new PR with your commited changes. Once you create the PR, the bot should automatically run.
+![add new pull request image](image.png)
+
+now the bot should be activated on Github actions
+![action page](image-2.png)
+![bot is processing the review](image-1.png)
+
+yay! the bot has given me a review on my PR. review the pr and make sure it make sense with your code. 
```

Comment:
This comment is asking for improvements on two aspects of the pull request: grammar and tone. The reviewer wants to ensure that the language used in the pull request is clear, concise, and professional. Improving grammar can help to avoid misunderstandings and ensure that the message is conveyed effectively. Improving tone can help to create a positive and respectful atmosphere in the collaboration process, which can lead to better communication and more productive discussions.

To address this comment, the author of the pull request could consider the following steps:

1. Review the pull request for grammatical errors, such as misspellings, incorrect verb tenses, and subject-verb agreement. Use a grammar checker tool, such as Grammarly, to help identify any errors.
2. Consider the tone of the language used in the pull request. Is it respectful and professional? Avoid using negative or confrontational language, and instead focus on providing constructive feedback and suggestions for improvement.
3. Revise the pull request to address any grammatical errors and improve the tone. Consider asking a colleague or mentor to review the revised pull request to ensure that it meets the desired standards for grammar and tone.
4. Respond to the reviewer's comment to let them know that the changes have been made. Thank the reviewer for their feedback and express appreciation for their help in improving the pull request.

By addressing this comment and making improvements to grammar and tone, the author of the pull request can help to create a more positive and productive collaboration experience for all parties involved.